build.sbt

```diff
@@ -3,7 +3,7 @@ lazy val scala211 = "2.11.12"
 
 ThisBuild / scalaVersion := scala212
 
-val scalapbVersion = "0.8.3"
+val scalapbVersion = "0.9.0-M6"
```

Comment:
Is there a good way to put this somewhere so it doesn't have to be in `build.sbt` and `plugins.sbt`?

Reply:
I think you could pull it out into a `projects/versions.scala`:
```
// project/versions.scala
object versions {
  val scalapb = "0.9.0-M6"
}

// use `versions.scalapb` in build.sbt and plugins.sbt
```

Reply:
Went with a different approach that was already in the `build.sbt` file